test/synapse/2019-06-01/azartifacts/zz_generated_pollers.go

```diff
@@ -22,6 +23,42 @@ type CreateDataFlowDebugSessionResponsePoller interface {
 	FinalResponse(ctx context.Context) (CreateDataFlowDebugSessionResponseResponse, error)
 }
 
+type createDataFlowDebugSessionResponsePoller struct {
+	pt *azcore.LROPoller
+}
+
+func (p *createDataFlowDebugSessionResponsePoller) Done() bool {
+	return p.pt.Done()
+}
+
+func (p *createDataFlowDebugSessionResponsePoller) Poll(ctx context.Context) (*http.Response, error) {
+	return p.pt.Poll(ctx)
+}
+
+func (p *createDataFlowDebugSessionResponsePoller) FinalResponse(ctx context.Context) (CreateDataFlowDebugSessionResponseResponse, error) {
+	respType := CreateDataFlowDebugSessionResponseResponse{CreateDataFlowDebugSessionResponse: &CreateDataFlowDebugSessionResponse{}}
```

Comment:
A comment for a separate discussion at a different time...but I dont like the envelope ending with ResponseResponse.

Reply:
Yes I agree.  I had brought this up with Jeff last week I think.